### DISCUSSION TITLE
Generate barcode

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3577] [Help] [answered]
Hi, can someone show me an example how to generate barcode in BB? Or anyway to import external svelte barcode component into BB?

**Comment by shogunpurple** [comment]
Hi @ryzam - you cannot do this right now. It's something that has been requested but not on the immediate roadmap. We'll possibly look at it in the coming months.

**Reply by igbojionu** [reply]
can this be done through api ?

**Reply by shogunpurple** [reply]
No - it can't. This is a separate component altogether, and is on the roadmap for the coming months.

**Comment by cfepdan** [comment]
FWIW, this could be done through an embed block and something like [this](https://gist.githubusercontent.com/Sahero/a04955d67852b8cf53db4c77edff91d3/raw/abdd776819af7121ac9caea7600d254c6bb3a21f/qrcode.html)

**Comment by gwlewis** [comment]
Just registering my interest here as well for the ability to generate a barcode.

**Comment by alexcmartins** [comment]
I'm interested in using a barcode generator

**Reply by ConorWebb96** [reply, marked as answer]
Hey @alexcmartins,

I think i added barcode generating functionality to this custom [plugin](https://github.com/ConorWebb96/bb-qr-generator). Let me know if this works for you.
